build-system/tasks/process-3p-github-pr.js

```diff
@@ -208,7 +208,7 @@ function getPullRequestFiles(pr) {
 /**
  * Determine the type of a give pull request
  * @param {?Array<!Object>} files
- * @return {*} TODO(#23582): Specify return type
+ * @return {number|undefined}
```

Comment:
shouldnt it be number or null? (not undefined)

Reply:
Good catch. It should be `{number|null|undefined}`, since there are three possible return values. Fixed.